spec/common/modules/shared_helper.js

```diff
@@ -90,7 +90,15 @@ define(['spec/common/modules/testapp_module', 'spec/common/modules/client_module
 
 		/* testFn is assumed to be a function of realtimeOptions that returns a nodeunit test */
 		function testOnAllTransports(exports, name, testFn, excludeUpgrade) {
-			utils.arrForEach(availableTransports, function(transport) {
+			/* Don't include jsonp if possible. Why? Because you can't abort requests. So recv's
```

Comment:
In `spec/browser/simple.test.js` (same as before I added the `testOnAllTransports` helper)